argilla-server/src/argilla_server/bulk/records_bulk.py

```diff
@@ -29,6 +29,8 @@
 )
 from argilla_server.api.schemas.v1.responses import UserResponseCreate
 from argilla_server.api.schemas.v1.suggestions import SuggestionCreate
+from argilla_server.api.webhooks.v1.enums import RecordEvent
+from argilla_server.api.webhooks.v1.records import notify_record_event
```

Comment:
I would add an alias to explicitly see that it's version 1:

```suggestion
from argilla_server.api.webhooks.v1.records import notify_record_event as notify_record_event_v1
```